tests/unit/services/unified-event-handler-test.js

```diff
@@ -28,6 +28,20 @@ test('register binds callback to the event on the specified target', function(as
   service.unregister('window', 'scroll', callbackStub);
 });
 
+test('unregisters event listeners when service is destroyed', function(assert) {
+  let done = assert.async();
+  let callbackStub = sandbox.stub();
+  service.register('window', 'scroll', callbackStub);
```

Comment:
Let's update this test to have multiple things registered, to make sure we're unregistering all the handlers.